Bottom: 1835ef413da81a4c084876f98e200094bb772503
Top:    3af97365c87b50f40f1aa6951332ccd5e74e39fa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 23:36:54 +0000

Refresh of hid-gtk-make-benchmark-impleme

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index c77faaf..0b84548 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1562,7 +1562,8 @@ Benchmark (int argc, char **argv, int x, int y)
   time (&start);
   do
     {
-      hid_expose_callback (&ghid_hid, &region, 0);
+      gdk_window_invalidate_rect (gport->drawing_area->window, NULL, 1);
+      gdk_window_process_updates (gport->drawing_area->window, FALSE);
       gdk_display_sync (display);
       time (&end);
       i++;
